x/shared/session.go

```diff
@@ -5,6 +5,10 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// closeBlockOffset is the offset to subtract from the earliest supplier commit
+// height to ensure that the supplier has enough time to commit the claim/proof.
+const closeBlockOffset = 1
```

Comment:
I ran a small program:

```
func main() {
	claimOpenHeight := 10
	claimWindow := 5
	for i := range 20 {
		result := claimOpenHeight + (i % claimWindow)
		fmt.Println(result)
	}
}
```

Which outputted things in `[10, 14]` INCLUSIVE.

Am I correct in understanding that the way this is currently working is that sometimes the supplier only has 1 block between "earliest claim commit height" and "claim window close" during which they have to do this?

If this is the case, then:
1. We can't and should not bandaid this but rather have a proper solution
2. If my understanding of (1) is not correct, please elaborate and help me understand

Reply:
Rought notes from convo with @red-0ne :

10: theoretical earliest
11: potential earliest
12: potential earliest
13: potential earliest
14: potential earliest
15: absolute latest

Situation: SUpplier 10:
Earliest: 14
Aboslute latest: 15

Assume 1 min block times.

If supplier downtime (e.g. server is down) for 61 seconds -> loses all money.